CONFIGURATION_OF_GOOGLE_SERVICE_ACCOUNT.md

```diff
@@ -1,22 +1,54 @@
-# Configuring google service account
+# Configuring Google Service Account
 
-This guide is simplified. For further information; look at googles documentation or find dedicated examples online.
+> Note: This guide is simplified. For further information look at Google's documentation or find dedicated examples online.
+
+A Google Service Account must be created / configured in a Google Cloud Project. There are two approaches to achieve
```

Comment:
IMO at the moment it's unclear which the two options are as steps and approaches share the same typography (h2).

Reply:
I wasn't quite happy with this approach but didn't see an obvious solution, therefore I decided to wait for your review. I'll try again :)